python/deltalake/table.py

```diff
@@ -89,7 +90,7 @@ def __init__(
         """
```

Comment:
Will want to document this parameter:

```suggestion
        :param without_files: If True, will load table without tracking files. Some 
          append-only applications might have no need of tracking any files. So, the
          DeltaTable will be loaded with a significant memory reduction.
        """
```